portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Yuyan!</h1>");
```

Comment:
It doesn't look like you wired up the JavaScript code to actually call the servlet?

Reply:
I have not done the js yet. This is only for step 1 of the server tutorial, simply testing that servlet connection is good. But I'm working on the js stuff during later steps.

Reply:
Ah. OK.